sams/sampler/FSStats.py

```diff
@@ -97,7 +97,10 @@ def do_sample(self):
     def sample(self):
         logger.debug("sample()")
         if self.fsstat:
-            self.store(self.fsstat.sample())
+            fsstat_sample = self.fsstat.sample()
+            self._most_recent_sample = self._sample_wrapping(fsstat_sample)
```

Comment:
```suggestion
            self._most_recent_sample = self._storage_wrapping(fsstat_sample)
```

Just caught this naming error, didn't notice that one earlier because I don't think we use this module locally.